e2e/__tests__/testEnvironmentCircus.test.ts

```diff
@@ -11,32 +11,85 @@ import runJest from '../runJest';
 skipSuiteOnJasmine();
 
 it('calls testEnvironment handleTestEvent', () => {
```

Comment:
could we add a new test rather than modifying the old one?

Reply:
sure, I'll do that soon